src/matcha_ml/cli/cli.py

```diff
@@ -124,8 +124,6 @@ def destroy(
 
     if full:
         remote_state_manager.deprovision_state_storage()
```

Comment:
Nit: `deprovision_remote_state` instead of `deprovision_state_storage` for consistency and also rest function if they mention state_storage in their name should be changed, as we are using `RemoteStateTemplate` and `RemoteStateRunner`.

Reply:
It's not part of this PR but `provision_state_storage` to `provision_remote_state`?

Reply:
@osw282 can you also have a look at above change if it's not a big change.